deploy/crds/broker_activemqartemis_crd.yaml

```diff
@@ -17,9 +17,570 @@ spec:
+                          expiryQueueSuffix:
+                            description: the suffix to use for auto-created expiry queues
+                            type: string
+                          expiryDelay:
+                            description: >-
+                              Overrides the expiration time for messages using the default value for expiration time. "-1"
+                              disables this setting.
+                            type: integer
+                          minExpiryDelay:
+                            description: Overrides the expiration time for messages using a lower value. "-1" disables this setting.
+                            type: integer
+                          maxExpiryDelay:
+                            description: Overrides the expiration time for messages using a higher value. "-1" disables this setting.
+                            type: integer
+                          redeliveryDelay:
+                            description: the time (in ms) to wait before redelivering a cancelled message.
+                            type: integer
+                          redeliveryDelayMultiplier:
+                            description: multiplier to apply to the redelivery-delay
+                            type: number
+                          redeliveryCollisionAvoidanceFactor:
+                            description: factor by which to modify the redelivery delay slightly to avoid collisions
+                            type: number
+                          maxRedeliveryDelay:
+                            description: Maximum value for the redelivery-delay
+                            type: integer
+                          maxDeliveryAttempts:
+                            description: how many times to attempt to deliver a message before sending to dead letter address
+                            type: integer
+                          maxSizeBytes:
+                            description: >-
+                              the maximum size in bytes for an address. -1 means no limits. This is used in PAGING, BLOCK and
+                              FAIL policies. Supports byte notation like K, Mb, GB, etc.
+                            type: string
+                          maxSizeBytesRejectThreshold:
+                            description: >-
+                              used with the address full BLOCK policy, the maximum size in bytes an address can reach before
+                              messages start getting rejected. Works in combination with max-size-bytes for AMQP protocol only.
+                              Default = -1 (no limit).
+                            type: integer
+                          pageSizeBytes:
+                            description: >-
+                              The page size in bytes to use for an address. Supports byte notation like K, Mb,
+                              GB, etc.
+                            type: integer
```

Comment:
@gaohoward This is still out of sync with the below data struct. We need to be sure we resolve the problem and make it the type it should be, and then determine any potential mitigation that is required inside the operator to deal with the change from v2alpha3 to v2alpha4.